templates/includes/sidebar.html

```diff
@@ -0,0 +1,8 @@
+<div>
+  <ol>
+    <li><a href="#">Home page</a></li>
+    <li><a href="#">Manufacturers</a></li>
+    <li><a href="#">Cars</a></li>
+    <li><a href="#">Drivers</a></li>

```

Comment:
The links in the sidebar are currently using placeholders (`#`). Consider updating these to point to the actual URLs within your application, using Django's `{% url %}` template tag if necessary.